doc/docker.md

```diff
@@ -0,0 +1,27 @@
+## How to install and use Docker for building
+
+Install Docker according to the following manuals:
+
+https://docs.docker.com/engine/install/ubuntu/
+
+https://docs.docker.com/engine/install/linux-postinstall/
+
+Pay attention to the section "Manage Docker as a non-root user", as it is critical
+for the proper usage of the container.
+
+We assume that you have already cloned the repo with sources and the `doc/` folder,
+which contains `Dockerfile` and `run_docker.sh`.
```

Comment:
it doesn't contain run_docker.sh anymore

Reply:
Thanks! Fixed.